docs/flow/modin/experimental/engines/omnisci_on_ray/frame/index.rst

```diff
@@ -0,0 +1,45 @@
+OmniSciOnRay Frame Implementation
+=================================
+
+Modin implements ``Frame``, ``PartitionManager``, ``AxisPartition`` and ``Partition`` classes
```

Comment:
Probably, we need to refactor OmniSciOnRay implementation for the release 0.11. I mean removing AxisPartition (because we don't use it at all), getting rid of Ray (if we really want it), restructuring directories, etc.